tools/sil-func-extractor/SILFunctionExtractor.cpp

```diff
@@ -256,7 +256,7 @@ int main(int argc, char **argv) {
 
   serialization::ExtendedValidationInfo extendedInfo;
   llvm::ErrorOr<std::unique_ptr<llvm::MemoryBuffer>> FileBufOrErr =
-      Invocation.setUpInputForSILTool(InputFilename, ModuleName, true,
+      Invocation.setUpInputForSILTool(InputFilename, ModuleName, true, false,
```

Comment:
Mind using comments to describe what each boolean literal means? `/*PrimaryFile*/false`

(Alternately, you could `enum class`-ify these parameters.)

Reply:
Done